src/qt/bitcoingui.cpp

```diff
@@ -16,6 +16,7 @@
 #include <qt/optionsmodel.h>
 #include <qt/platformstyle.h>
 #include <qt/rpcconsole.h>
+#include <qt/transactionrecord.h>
```

Comment:
Maybe the constant is in the wrong place?

Reply:
Yes, this place seems inappropriate. The reason was to avoid circular dependencies.
@promag Could you point out a more appropriate place?